spec/rails_helper.rb

```diff
@@ -3,12 +3,7 @@
 require 'spec_helper'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
-
-# Add additional requires below this line. Rails is not loaded until this point!
-
-require 'factory_girl'
-FactoryGirl.definition_file_paths = [File.expand_path('../factories', __FILE__)]
-FactoryGirl.find_definitions
+require 'factory_girl_rails'
```

Comment:
Yep, `factory_girl_rails` handles all of this for us. Not sure why we weren't using it before..